docs/experiments/categories/gcp/gcp-vm-instance-stop/gcp-instance.yaml

```diff
@@ -0,0 +1,26 @@
+## details of the gcp instance
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
```

Comment:
Can we keep the name as `gcp-vm-chaos`, as that is what we have mentioned in the docs as default?